### ISSUE TITLE
Move row numbers and  justifications outside scrollable box

### ISSUE THREAD

**Issue by McTano** [McTano/Rudolf#78]
Since the row number and justification are relevant no matter which sub-branch you're working on, I was thinking maybe they should be moved outside the scrollable container so that they can always be seen, even if you're looking at the middle part of the tree.

Just a thought. Anyone have opinions on this?

**Comment by mbecker20** [comment]
Maybe the justifications could act as they do now when they are in view, but as they are scrolled horizontally out of the tree window, they "catch" on the edge, and the rest of the tree keeps scrolling underneath them. I think this could be better because the justifications still have to scroll vertically to stay horizontally aligned with the appropriate tree row, and being inside the scroll window gives the overlapping content a clean edge when it is inserted as a widget ( otherwise you have Rudolf content overflowing outside of it's widget bounds

**Comment by McTano** [comment]
Yeah, That would be better. I wasn't thinking about vertical scrolling.
